### PR TITLE
Add io.github.GoldenGnu.jEveAssets

### PR DESCRIPTION
**Pull request by Nathan-LS**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. [Flatpak distribution of jEveAssets #434 ](https://github.com/GoldenGnu/jeveassets/issues/434)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *I scaled the app icon from 64x64 to fulfill the 128x128 requirement. Metainfo was not submitted upstream as the changelog is specific to flatpak with autoupdating static data functionality disabled.* 

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: io.github.GoldenGnu.jEveAssets.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.GoldenGnu.jEveAssets</id>
+    <launchable type="desktop-id">io.github.GoldenGnu.jEveAssets.desktop</launchable>
+    <name>jEveAssets</name>
+    <developer_name>GoldenGnu</developer_name>
+    <summary>jEveAssets is an out-of-game asset manager for Eve-Online, written in Java.</summary>
```
Comment (hfiguiere) [inline io.github.GoldenGnu.jEveAssets.metainfo.xml:7]:
```suggestion
    <summary>jEveAssets is an out-of-game asset manager for Eve-Online</summary>
```

File: jEveAssets [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+# make a symnlink so data is persisted under .var/app/io.github.GoldenGnu.jEveAssets/data/.jeveassets as jEveAssets hardcodes data to be under ~/.jeveassets
+ln -s "$XDG_DATA_HOME/.jeveassets" "$HOME/.jeveassets"
```
Comment (hfiguiere) [inline jEveAssets:4]: you could just `--persist=.jeveassets` instead.

File: jEveAssets [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+# make a mount directory for saving filters, accounts, etc from the application instead of giving full permission to the user's home directory
+mkdir -p "$XDG_DATA_HOME/jEveAssets_mnt"
+ln -s "$XDG_DATA_HOME/jEveAssets_mnt" "$HOME/jEveAssets_mnt"
```
Comment (hfiguiere) [inline jEveAssets:5]: this can be persisted too.

Comment (Nathan-LS) [reply jEveAssets]: Done.

**Comment by Nathan-LS** [comment]
bot, build io.github.GoldenGnu.jEveAssets

**Review by bbhtt — COMMENTED** [review]
Can you put icon, desktop file and appdata upstream? 

Also, the icon is a bit blurry, not sure if that's fixable.

**Comment by Nathan-LS** [comment]
> Can you put icon, desktop file and appdata upstream? 
> 
> Also, the icon is a bit blurry, not sure if that's fixable.

I will work on getting those submitted to upstream.

**Pull request closed without merging**